docs/sections/header.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/main.js"></script>
+	</head>
+	<body>
+        <div class="container">
+            <h1 class="mt-1">Header</h1>
+        </div>
+
+        <div class="announcement-bar bg-primary-light text-center w-100">
+            <div class="container">
+                <a href="#"><strong>Up to 25% off + Free Gift worth $25.40</strong></a>
+            </div>
+        </div>
+        <header class="navbar navbar-expand-lg px-1 px-lg-0">
+            <div class="container">
+                <button class="navbar-toggler text-left col-2 px-0" type="button" data-trigger="#navbar_main">
+                    <span class="navbar-toggler-icon d-inline-block"></span>
+                </button>
+
+                <div class="navbar-collapse collapse w-100 px-0">
+                    <ul class="navbar-nav align-items-center">
+                        <li class="nav-item dropdown position-static">
+                            <a class="nav-link dropdown-toggle text-white btn btn-primary" href="#" aria-haspopup="true" aria-expanded="false" data-target="megaMenuDropdown">Shop</a>
+                            <div id="megaMenuDropdown" class="dropdown-menu dropdown-menu--mega-menu mt-0 w-100 d-block">
+                                <div class="container d-block">
```

Comment:
these can be merged

Reply:
@foria I tried to merged this, but this will causing mega menu width not 100%, I think this will be issue when megamenu appear in top of hero banner

http://prntscr.com/10taz8q